redaxo/src/addons/backup/lib/backup.php

```diff
@@ -21,6 +21,21 @@ public static function getDir()
         return $dir;
     }
 
+    /**
+     * @param self::IMPORT_* $importType
+     */
+    public static function validateFilename($importType, string $filename): bool
```

Comment:
Ich würde da trotzdem auch den nativen Typ angeben:

```suggestion
    public static function validateFilename(int $importType, string $filename): bool
```

Reply:
Hmm und als Name vielleicht eher was mit `is`-Präfix?

```suggestion
    public static function isFilenameValid(int $importType, string $filename): bool
```